### DISCUSSION TITLE
Idea / Feature request: Disable timestamps in ansible output

### DISCUSSION THREAD

**Discussion by dwedia** [semaphoreui/semaphore#2639] [Ideas]
While i love SemaphoreUI, a feature I really miss is the option to disable the timestamps in the ansible output.

![semaphoreTimestamps](https://github.com/user-attachments/assets/de0a7b1a-6165-49f1-ac9c-c0929fe09af1)

The SemaphoreUI already shows me when I started a job, and how long it took. The timestamp on every line is nice for debugging, but redundant for normal day to day use.

If you could find a way to add a checkbox or setting where we can choose if those timestamps are enabled or not, it would be amazing =)

**Comment by svennd** [comment]
Yes or beeing able to export, would be nice https://github.com/semaphoreui/semaphore/issues/2118 perhaps its going to become a pro feature at some point. (see https://github.com/semaphoreui/semaphore-pro )
